docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#network-v1-config-openshift-io
+
+### Sample data
+- [docs/insights-archive-sample/config/network.json](./insights-archive-sample/config/network.json)
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/network.go#L50
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#network-v1-config-openshift-io
+### Location in archive
+- `config/network.json`
 
-* Location in archive: config/network/
-* See: docs/insights-archive-sample/config/network
-* Id in config: clusterconfig/networks
+### Config ID
+`clusterconfig/networks`
+
+### Released version
+- 4.2.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## ClusterOAuth
 
-fetches the cluster OAuth - the OAuth with name cluster.
+Collects the cluster OAuth - the OAuth with name cluster.
+
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/oauth.go#L50
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#oauth-v1-config-openshift-io
+
+### Sample data
+- [docs/insights-archive-sample/config/oauth.json](./insights-archive-sample/config/oauth.json)
+
+### Location in archive
+- `config/oauth.json`
+
+### Config ID
+`clusterconfig/oauths`
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/oauth.go#L50
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#oauth-v1-config-openshift-io
+### Released version
+- 4.2.0
 
-* Location in archive: config/oauth/
-* See: docs/insights-archive-sample/config/oauth
-* Id in config: clusterconfig/oauths
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## ClusterOperatorPodsAndEvents
 
-collects information about pods
+Collects information about pods
 and events from namespaces of degraded cluster operators. The collected
 information includes:
 
 - Definitions for non-running (terminated, pending) Pods
 - Previous (if container was terminated) and current logs of all related pod containers
 - Namespace events
 
-  - Location of pod definitions: config/pod/{namespace}/{pod}.json
-  - Location of pod container current logs:
-    config/pod/{namespace}/logs/{pod}/{container}_current.log
-  - Location of pod container previous logs:
-    config/pod/{namespace}/logs/{pod}/{container}_previous.log
-  - Location of events in archive: events/
-  - Id in config: clusterconfig/operators_pods_and_events
-  - Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/pod](./insights-archive-sample/pod)
+- [docs/insights-archive-sample/events](./insights-archive-sample/events)
+
+### Location in archive
+- `config/pod/{namespace}/{pod}.json`
+- `events/`
```

Comment:
What is the structure of the folder? There are other gatherers that put files to the folder. What makes sure the gatherers do not collide?

Reply:
Actually, judging by other gatherers, I assume it should look like this:

```suggestion
- `events/{namespace}.json`
```

Reply:
@jholecek-rh when you said: "What makes sure the gatherers do not collide?" - I was pointing this for @tremes on our channel, probably it will not be something frequent, but, IMHO, it is just a guest, we probably need to investigate it and make sure that collisions are not happening. There are other gathers that share the same path in the archive.

PS: this is just a note, I don't expect interaction about path collisions here, 'cause this is something that needs to be addressed in a separate task.